package.json

```diff
@@ -23,7 +23,9 @@
     "start:background": "npm run cli -- background",
     "pretest": "npm run lint && cross-env NODE_ENV=test npm run db:migrate:reset",
     "test": "jest --testEnvironment=node --runInBand",
-    "typeorm": "typeorm-ts-node-commonjs"
+    "typeorm": "typeorm-ts-node-commonjs",
+    "temporal:dev": "cross-env TEMPORAL_ADDRESS=0.0.0.0:7233 nodemon ./src/temporal/local.ts",
```

Comment:
we have our convention of `dev:background` and `start:background` can we stick to it please?

Reply:
Right. Let me update this quickly.